scalecube-configuration/src/test/java/io/scalecube/configuration/fixtures/ContainersConfigurationServiceFixture.java

```diff
@@ -0,0 +1,223 @@
+      e.printStackTrace();
+    }
+  }
+
+  private void setUpCouchbase() {
+    couchbaseContainer =
+        new CouchbaseContainer(COUCHBASE_DOCKER_IMAGE)
+            .withClusterAdmin(COUCHBASE_USERNAME, COUCHBASE_PASSWORD)
+            .withNetwork(Network.SHARED)
+            .withNetworkAliases(COUCHBASE_NETWORK_ALIAS)
+            .withCreateContainerCmdModifier(
+                cmd -> {
+                  cmd.withName(COUCHBASE_NETWORK_ALIAS);
+                  cmd.withPortBindings(PortBinding.parse(COUCHBASE_PORT + ":" + COUCHBASE_PORT));
+                });
+    couchbaseContainer.start();
+    couchbaseContainer.initCluster();
+    try {
+      couchbaseContainer.callCouchbaseRestAPI("/settings/indexes", "storageMode=forestdb");
+    } catch (IOException e) {
+      LOGGER.warn("Couchbase set up issues", e);
+    }
+
+    createBucket(couchbaseContainer, ORGANIZATIONS_BUCKET, COUCHBASE_PASSWORD);
+    createBucket(couchbaseContainer, CONFIGURATIONS_BUCKET, COUCHBASE_PASSWORD);
+  }
+
+  private void setUpVault() {
+    vaultContainer =
+        new VaultContainer<>(VAULT_DOCKER_IMAGE)
+            .withVaultPort(VAULT_PORT)
+            .withVaultToken(VAULT_TOKEN)
+            .withSecretInVault(VAULT_SECRETS_PATH,
+                "couchbase.hosts=" + COUCHBASE_NETWORK_ALIAS,
+                "couchbase.username=" + COUCHBASE_USERNAME,
+                "couchbase.password=" + COUCHBASE_PASSWORD)
+            .withNetwork(Network.SHARED)
+            .withNetworkAliases(VAULT_NETWORK_ALIAS)
+            .withCreateContainerCmdModifier(
+                cmd -> {
+                  cmd.withName(VAULT_NETWORK_ALIAS);
+                  cmd.withPortBindings(PortBinding.parse(VAULT_PORT + ":" + VAULT_PORT));
+                })
+            .waitingFor(new LogMessageWaitStrategy().withRegEx("^.*Vault server started!.*$"));
+    vaultContainer.start();
+  }
+
+  private void setUpGateway() {
+    gatewayContainer =
+        new GenericContainer<>("scalecube/scalecube-services-gateway-runner:2.5.10")
+            .withExposedPorts(WS_GATEWAY_PORT, HTTP_GATEWAY_PORT, RS_GATEWAY_PORT)
+            .withNetwork(Network.SHARED)
+            .withNetworkAliases(GATEWAY_NETWORK_ALIAS)
+            .withCreateContainerCmdModifier(
+                cmd -> {
+                  cmd.withName(GATEWAY_NETWORK_ALIAS);
+                  cmd.withPortBindings(
+                      PortBinding.parse(WS_GATEWAY_PORT + ":" + WS_GATEWAY_PORT),
+                      PortBinding.parse(HTTP_GATEWAY_PORT + ":" + HTTP_GATEWAY_PORT),
+                      PortBinding.parse(RS_GATEWAY_PORT + ":" + RS_GATEWAY_PORT)
+                  );
+                })
+            .waitingFor(new HostPortWaitStrategy());
+    gatewayContainer.start();
+  }
+
+  private void setUpOrganizationService(Map<String, String> env) {
+    env.put("JAVA_OPTS", "-Dio.scalecube.organization.seeds=" + GATEWAY_NETWORK_ALIAS
+        + ":4801 -DmockTokenVerifier=true");
```

Comment:
Let's avoid using such approach: adding some mocks (moreover, adding mocks to production code) to repository of organization service just to use it for tests in another projects (configuration service).

Set up organization service without Docker container and mock anything you need.